src/web-animations-next-animation.js

```diff
@@ -63,6 +63,8 @@
         return;
       for (var i = 0; i < this.effect.children.length; i++) {
         this.effect.children[i].animation = animation;
+      }
+      for (var i = 0; i < this._childAnimations.length; i++) {
```

Comment:
It's possible for them to be different lengths, but not when this is called. I have changed it back.